README.md

```diff
@@ -25,3 +25,36 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+If the algorithm goes to the base case, it will have a time complexity of T(1)=1. Splitting the array then has a constant time complexity (T(n)=1). After that, it makes 3 recursive calls on a third of the input and adds them so the time complexity will be T(n) = 3T(n/3). And adding the sum of the arrays also has a time complexity of T(1)=1. 
+
+This means that the recurrence relation is T(n) = 1 when n<=1. T(n) = 3T(n/3) when n>1.
```

Comment:
You need to take the complexity of combining the results into account. You can't ignore it because it's a constant.